src/test/unit/profile-data.test.js

```diff
@@ -843,17 +843,27 @@ describe('getTimingsForPath in a non-inverted tree', function() {
     const timings = getTimingsForPath([A]);
     expect(timings).toEqual({
       forPath: {
-        selfTime: { value: 0, breakdownByImplementation: null },
+        selfTime: {
+          value: 0,
+          breakdownByImplementation: null,
+          breakdownByCategory: null,
+        },
         totalTime: {
           value: 5,
           breakdownByImplementation: { native: 2, baseline: 1, ion: 2 },
+          breakdownByCategory: [1, 3, 1], // [Idle, Other, Layout]
```

Comment:
Please provide variable bindings for the numbers. This will help with the readability.
```
const IDLE = 1;
const OTHER = 3;
```

Reply:
Used `Array(categories.length).fill(0)` as we discussed